### PR TITLE
Removed unreachable and unused/dead code

### PR DESCRIPTION
**Pull request by venkateshpitta**
## Code contributions to the Bitmessage project

- try to explain what the code is about
- try to follow [PEP0008](https://www.python.org/dev/peps/pep-0008/)
- make the pull request against the ["v0.6" branch](https://github.com/Bitmessage/PyBitmessage/tree/v0.6)
- it should be possible to do a fast-forward merge of the pull requests
- PGP-sign the commits included in the pull request
- You can get paid for merged commits if you register at [Tip4Commit](https://tip4commit.com/github/Bitmessage/PyBitmessage)

If for some reason you don't want to use github, you can submit the patch using Bitmessage to the "bitmessage" chan, or to one of the developers.
## Translations

For helping with translations, please use [Transifex](https://www.transifex.com/bitmessage-project/pybitmessage/). There is no need to submit pull requests for translations.
For translating technical terms it is recommended to consult the [Microsoft Language Portal](https://www.microsoft.com/Language/en-US/Default.aspx).

### CODE REVIEW

**Comment by PeterSurda** [comment]
Hello,

a couple of comments.

- the pull request isn't against v0.6 branch, I actually removed some obsolete files in the meantime
- the commit isn't pgp signed. Please create a PGP key and upload it to your github account (under Settings). Then you can sign the commits with `git commit -s` or you can set the signing options in the git configuration for the project

**Comment by venkateshpitta** [comment]
Is anyone looking at this?  any comment is welcome

**Comment by PeterSurda** [comment]
Well I already made the organisational comments. Looking through the changes, some of it do look like obsolete code but aren't as they are a part of an object or a dictionary and without them the code would break.

**Comment by venkateshpitta** [comment]
Added .travis.yml.  Lacking tests, it fails.  Also, made a project specific requirements.txt to be used by Travis.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+language: python
+
+virtualenv:
+  system_site_packages: false # default, just making explicit
+
+python:
+  - "2.6"
```
Comment (PeterSurda) [inline .travis.yml]: Doesn't work on 2.6, needs at least 2.7.2 (possible 2.7.6 even).

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+setuptools==38.5.2
+numpy==1.13.3
+AppKit==0.2.8
```
Comment (PeterSurda) [inline requirements.txt]: I am not familiar with this.

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+PyQt4==4.11.4
+dialog==0.2.0
+gi==1.2
+msgpack_python==0.5.6
+prctl==1.0.1
+pybloom==1.1
+pybloomfiltermmap==0.3.15
```
Comment (PeterSurda) [inline requirements.txt]: Bloom filters are just for some experiments and aren't actually needed at the moment.

**Pull request closed without merging**